internal/controller/scheduler/scheduler.go

```diff
@@ -53,12 +54,13 @@ func (scheduler *Scheduler) RequestScheduling() {
 
 func (scheduler *Scheduler) schedulingLoopIteration() error {
 	affectedWorkers := map[string]bool{}
+
 	err := scheduler.store.Update(func(txn storepkg.Transaction) error {
 		vms, err := txn.ListVMs()
 		if err != nil {
 			return err
 		}
-		unscheduledVMs, workerToResources := processVMs(vms)
+		scheduledVMs, unscheduledVMs, workerToResources := processVMs(vms)
```

Comment:
"scheduled" VM in a "scheduling" loop kind of intends that it's being scheduled now when these are "already scheduled" VMs before. Seems naming can be a bit better.

What do you think of moving the health checking logic in a separate "healthcheckingLoopIteration"?

Reply:
Moved in https://github.com/cirruslabs/orchard/pull/70/commits/2a4e3af61a5e8fde5651da8f85fa3628dfacfaea.